docs/xks/operator-guide/operator-guide.md

```diff
@@ -272,7 +353,7 @@ To make sure that our developers don't notice when we change our the cluster we
 
 AZAD as we also call it, is a deployment that runs inside XKS and sits in front of the kubernetes API.
 
-We also supply a krew/kubectl plugin to make it easy for our developers to use AZAD.
+We also supplie a krew/kubectl plugin to make it easy for our developers to use AZAD.
```

Comment:
Spurious edit?

Reply:
Don't understand what you mean. But I fixed the spelling error :)